ansible/roles/jetson_services/files/rover-jetson-cameras.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=MRover Cameras
+
+[Service]
+EnvironmentFile=/etc/default/rover.env
+User={{ running_user }}
+ExecStart={{ workspace_dir }}/jarvis exec jetson_cameras_
```

Comment:
jetson_cameras, why the extra underscore :^)

Reply:
why must you tear me down